samples/pom.xml

```diff
@@ -34,6 +34,11 @@
 
   <name>Apache Apex Malhar (incubating) Samples</name>
 
+  <properties>
+    <semver.plugin.skip>true</semver.plugin.skip>
```

Comment:
It depends on the percentage of modules that semver should be enabled and disabled for. In Malhar, there are more modules where it needs to be disabled, it seems that repeating 
```
      <plugin>
        <groupId>com.github.siom79.japicmp</groupId>
        <artifactId>japicmp-maven-plugin</artifactId>
      </plugin>
```
only for modules where semver needs to be enabled will be more applicable in this case.